app/vep/models/client_model.py

```diff
@@ -1 +1,38 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional
 
+class FieldOptions:
+    GENCODE_LABEL: str = "GENCODE"
+    GENCODE_VALUE: str = "gencode_comprehensive"
```

Comment:
Sorry, I am very confused about the naming of these. Why are we calling them `GENCODE_LABEL` or `GENCODE_VALUE`. "Gencode" is the annotation provider just for human. This should probably called `TRANSCRIPT_SET_LABEL` and `TRANSCRIPT_SET_VALUE`; and it doesn't make sense to have default values for them, because they are genome-specific.

Reply:
It seems that this whole class is unnecessary.